test/drivers/test_suites/syscall_exit_suite/socketcall_x.cpp

```diff
@@ -193,9 +193,12 @@ TEST(SyscallExit, socketcall_connectX)
 	/* The client performs a `connect` so the client is the src. */
 	evt_test->assert_tuple_inet_param(2, PPM_AF_INET, IPV4_CLIENT, IPV4_SERVER, IPV4_PORT_CLIENT_STRING, IPV4_PORT_SERVER_STRING);
 
+	/* Parameter 3: fd (type: PT_FD) */
+	evt_test->assert_numeric_param(3, (int64_t)client_socket_fd);
```

Comment:
Awesome! I would about to open a PR for this as well.

Reply:
@Andreagit97 While at the tests, could you cherry pick [update(tests): correct snaplen HTTP checks on s390x ](https://github.com/hbrueckner/falcosecurity-libs/commit/557e56eba4a79710ffdf2f0ebcb654382667775c) to this series? It corrects an endianess issue in the modern/old BPF drivers when comparing `BPF_HTTP_*` constants against a character array.

Reply:
Sure I will add it! I was just investigating the same failing tests :laughing:

Reply:
@Andreagit97 Thanks!

/approve